source/_posts/2016-08-09-we-have-apps-now.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "We Have Apps Now"
+description: "A new subsystem that allows automations to be coded using Python"
+date: 2016-08-09 14:00:00 -0400
+date_formatted: "August 09, 2016"
+author: Andrew Cockburn
+comments: true
+categories: How-To
+---
+
+I have been working on a new subsystem to complement Home Assistant's Automation and Scripting components. `AppDaemon` is a python daemon that consumes events from Home Assistant and feeds them to snippets of python code called "Apps". An App is a Python class that is instantiated possibly multiple times from AppDaemon and registers callbacks for various system events. It is also able to inspect and set state and call services. The API provides a rich environment suited to home automation tasks that can also leverage all the power of Python.
+
+# Another Take on Automation
```

Comment:
You can add `# {% linkable_title Title %}` which would make it easier to reference to a section.